README.md

```diff
@@ -268,8 +279,8 @@ Number of Fisher Scoring iterations: 4
 In Julia this becomes
 ```julia
 julia> dobson = DataFrame(Counts = [18.,17,15,20,10,20,25,13,12],
-                          Outcome = gl(3,1,9),
-                          Treatment = gl(3,3))
+                          Outcome = categorical(repeat(1:3, outer=3)),
+                          Treatment = categorical(repeat(1:3, inner=3)))
 9x3 DataFrame
```

Comment:
That code shows as
```
9×3 DataFrame
│ Row │ Counts │ Outcome                          │ Treatment                        │
├─────┼────────┼──────────────────────────────────┼──────────────────────────────────┤
│ 1   │ 18.0   │ CategoricalValue{Int64,UInt32} 1 │ CategoricalValue{Int64,UInt32} 1 │
│ 2   │ 17.0   │ CategoricalValue{Int64,UInt32} 2 │ CategoricalValue{Int64,UInt32} 1 │
│ 3   │ 15.0   │ CategoricalValue{Int64,UInt32} 3 │ CategoricalValue{Int64,UInt32} 1 │
│ 4   │ 20.0   │ CategoricalValue{Int64,UInt32} 1 │ CategoricalValue{Int64,UInt32} 2 │
│ 5   │ 10.0   │ CategoricalValue{Int64,UInt32} 2 │ CategoricalValue{Int64,UInt32} 2 │
│ 6   │ 20.0   │ CategoricalValue{Int64,UInt32} 3 │ CategoricalValue{Int64,UInt32} 2 │
│ 7   │ 25.0   │ CategoricalValue{Int64,UInt32} 1 │ CategoricalValue{Int64,UInt32} 3 │
│ 8   │ 13.0   │ CategoricalValue{Int64,UInt32} 2 │ CategoricalValue{Int64,UInt32} 3 │
│ 9   │ 12.0   │ CategoricalValue{Int64,UInt32} 3 │ CategoricalValue{Int64,UInt32} 3 │
```
which can be avoided by having the categorical variables as `AbstractString`
```
julia> dobson = DataFrame(Counts = [18,17,15,20,10,20,25,13,12],
                          Outcome = categorical(repeat(string.(1:3), outer = 3)),
                          Treatment = categorical(repeat(string.(1:3), inner = 3)))
9×3 DataFrame
│ Row │ Counts │ Outcome │ Treatment │
├─────┼────────┼─────────┼───────────┤
│ 1   │ 18     │ 1       │ 1         │
│ 2   │ 17     │ 2       │ 1         │
│ 3   │ 15     │ 3       │ 1         │
│ 4   │ 20     │ 1       │ 2         │
│ 5   │ 10     │ 2       │ 2         │
│ 6   │ 20     │ 3       │ 2         │
│ 7   │ 25     │ 1       │ 3         │
│ 8   │ 13     │ 2       │ 3         │
│ 9   │ 12     │ 3       │ 3         │
```
You should probably have counts `[18,...]` such that it becomes `Vector{Int64}` rather than `Vector{Float64}` as counts is restricted to integers.

Reply:
dot in "18." removed. I do not know how it went in there 😄. As for showing `Outcome` and `Treatement` what version of Julia and DataFrames-ecosystem packages are you using? On my system it shows exactly as pasted to README.md.